LiteNetLib/NetPacket.cs

```diff
@@ -148,7 +148,8 @@ public bool FromBytes(byte[] data, int start, int packetSize)
             int headerSize = GetHeaderSize((PacketProperty) property);
 
             if (property > LastProperty || packetSize < headerSize ||
-               (fragmented && packetSize < headerSize + NetConstants.FragmentHeaderSize))
+               (fragmented && packetSize < headerSize + NetConstants.FragmentHeaderSize) ||
+               data.Length < start + packetSize)
```

Comment:
Better move this check to merged packet reading.

Reply:
there're two usages of this method and both are recycling packet if anything goes wrong... could you please point me to the better place as i'm not sure where's the right place to move this check to